ui/app/helpers/engines-display-name.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { ALL_ENGINES } from 'vault/utils/engines-display-data';
+
+export default function engineDisplayName(methodType: string) {
+  const displayName = ALL_ENGINES?.find((t) => t.type === methodType)?.displayName;
+  return displayName || methodType;
+}
```

Comment:
is `displayName` often the only key we need? Wondering if it'd be good to generalize this more to just return the object. Then in the template call `get` for whichever the desired parameter. This would be useful to get `icon` data for example if/when we migrate away from using getters in the model